prow/cmd/deck/static/pr-script.js

```diff
@@ -887,7 +910,49 @@ function createStatusHelp(title, content) {
     return helpIcon;
 }
 
-function createPRCardBody(pr, builds, queries) {
+/**
+ * Creates the branch conflict status.
+ * @param pr
+ * @param branchConflict
+ * @return {Element}
+ */
+function createBranchConflictStatus(pr, branchConflict) {
+    const statusContainer = document.createElement("DIV");
+    statusContainer.classList.add("status-container");
+    const status = document.createElement("DIV");
+    if (branchConflict) {
+        status.appendChild(createIcon("error", "", ["status-icon", "failed"]));
+        status.appendChild(
+            document.createTextNode(`Merging to branch ${pr.BaseRef.Name} is not configured`));
```

Comment:
I imagine this will send developers to the on call to ask them to configure the branch. Instead, let's be clear this is explicitly the desired behavior:

    "Merging into branch ${pr.baseRef.name} is currently forbidden"

Reply:
I don't think we should use this wording. The PR dashboard shows all of the author's PRs including PRs that are in repos not managed by Tide. These PRs are not forbidden from merging, they just won't be merged *automatically*.